webapp/src/webapp/connections/views/form/database.cljs

```diff
@@ -192,6 +196,7 @@
             :checked? access-mode-runbooks}
            {:name "connect"
             :label "Native"
+            :disabled? (= @connection-subtype "oracledb")
```

Comment:
Add a message also saying we don't support oracle at the side of the "Native" text so people can know about it